lisa/node.py

```diff
@@ -344,6 +368,31 @@ def close(self) -> None:
         for node in self._list:
             node.close()
 
+    def from_either(
+        self,
+        node_runbook: Union[schema.LocalNode, schema.RemoteNode],
+        logger_name: str = "node",
+        base_log_path: Optional[Path] = None,
+    ) -> Optional[Node]:
+        if isinstance(node_runbook, schema.LocalNode):
+            return self.from_local(node_runbook, logger_name, base_log_path)
+        else:
+            assert isinstance(
+                node_runbook, schema.RemoteNode
+            ), f"actual: {type(node_runbook)}"
+            return self.from_remote(node_runbook, logger_name, base_log_path)
+
+    # Keep non-SUT node always at the end. We don't envision
```

Comment:
This change is a little bit hacking. Check previous comments, how about use a subclass, so the non-sut node can be separately.

Reply:
I've said why it's not a good idea, please check out.